### ISSUE TITLE
Unable to generate icon action/settings

### ISSUE THREAD

**Issue by Vlado24** [Dhaval2404/material-icon-generator-plugin#4]
Hi,

I'm using Android Studio 4.2 Beta 2 with your plugin (version 1.2) and I'm not able to generate icon `action/settings`. 

Here is stacktrace with exception:

```
java.lang.IllegalStateException: java.io.FileNotFoundException: Source '/private/var/folders/22/_jjg0tks4dn_cpzspf4krkcr0000gn/T/material_icon_generator/res/drawable/baseline_settings_24.xml' does not exist
	at com.github.dhaval2404.material_icon_generator.widget.MaterialDesignIconGenerateDialog.createVectorIcon(MaterialDesignIconGenerateDialog.java:569)
	at com.github.dhaval2404.material_icon_generator.widget.MaterialDesignIconGenerateDialog.createIcons(MaterialDesignIconGenerateDialog.java:508)
	at com.github.dhaval2404.material_icon_generator.widget.MaterialDesignIconGenerateDialog.create(MaterialDesignIconGenerateDialog.java:490)
	at com.github.dhaval2404.material_icon_generator.widget.MaterialDesignIconGenerateDialog.doOKAction(MaterialDesignIconGenerateDialog.java:483)
	at com.intellij.openapi.ui.DialogWrapper$OkAction.doAction(DialogWrapper.java:1881)
	at com.intellij.openapi.ui.DialogWrapper$DialogWrapperAction.actionPerformed(DialogWrapper.java:1834)
```

**Comment by kallydev** [comment]
I also encountered this problem and found the download link of the icon.

https://fonts.gstatic.com/s/i/materialicons/settings/v1/black-android.zip

After comparing with other icons, I determined that this error is caused by the lack of drawable folders for the settings icon.
Fortunately, the missing folder exists in the GitHub repository. 

https://github.com/google/material-design-icons/tree/master/android/action/settings/materialicons/black/res/drawable

Because all changes can be tracked on GitHub, please consider migrating from Google Fonts to GitHub. @Dhaval2404

**Comment by Dhaval2404** [comment]
@Vlado24 issue has been fixed in v1.3. Thank you so much for reporting the issue.
